components/Cards/card.html

```diff
@@ -0,0 +1,262 @@
+              <div class="card-main-btn vertical-flex">
+                <div><a href="" class="remove-deco text-small">Expand for more details</a></div>
+                <div><a href="" class="remove-deco text-small">Add to cart</a></div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <h1>Card with text overlay</h1>
+
+        <div class="container-card">
+          <div class="card vertical-flex card-vertical">
+            <div class="card-img">
+              <img src="/components/images/sofa.svg" alt="">
+            </div>
+            <div class="card-main vertical-flex">
+              <div class="card-main-title vertical-flex">
+                <div class="product-title"><b>Sofa</b></div>
+                <div class="product-price">Rs. 3999</div>
+              </div>
+              <div class="card-main-desc text-gray text-small">Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusamus, labore!</div>
+              <div class="card-main-btn vertical-flex">
+                <div><a href="" class="remove-deco text-small">Expand for more details</a></div>
+                <div><a href="" class="remove-deco text-small">Add to cart</a></div>
+              </div>
+            </div>
+            <div class="card-text-overlay">Out of Stock</div>
+          </div>
+
+          <div class="card horizontal-flex card-horizontal">
+            <div class="card-img">
+              <img src="/components/images/sofa.svg" alt="">
+            </div>
+            <div class="card-main vertical-flex">
+              <div class="card-main-title vertical-flex">
+                <div class="product-title"><b>Sofa</b></div>
+                <div class="product-price">Rs. 3999</div>
+              </div>
+              <div class="card-main-desc text-gray text-small">Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusamus, labore!</div>
+              <div class="card-main-btn vertical-flex">
+                <div><a href="" class="remove-deco text-small">Expand for more details</a></div>
+                <div><a href="" class="remove-deco text-small">Add to cart</a></div>
+              </div>
+            </div>
+            <div class="card-text-overlay">Out of Stock</div>
+          </div>
+        </div>
+
+        <h1>Text Only card</h1>
+
+        <div class="container-card">
+          <div class="card vertical-flex card-vertical text-card">
+            <div class="text-card-title">Title</div>
+            <div class="text-card-desc">Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur, ipsam excepturi, fugit molestias est repellat rem exercitationem, ab perferendis tempore repellendus iusto totam qui optio illo rerum iste eveniet placeat!
+            </div>
+            <button class="text-card-button">Read More</button>
+          </div>
+        </div>
+
+        <iframe

```

Comment:
It's my opinion, instead of adding the entire code snippet at once you can add below the respective component